app/src/main/kotlin/com/hippo/ehviewer/ui/screen/DownloadsScreen.kt

```diff
@@ -150,14 +150,16 @@ fun DownloadsScreen(navigator: DestinationsNavigator) {
     val view = LocalView.current
     val allName = stringResource(R.string.download_all)
     val defaultName = stringResource(R.string.default_download_label_name)
-    val title = stringResource(R.string.scene_download_title, if (label == "") allName else label ?: defaultName)
+    val title = stringResource(
+        R.string.scene_download_title,
+        with(filterState) { if (label == "") allName else label ?: defaultName },
+    )
     val hint = stringResource(R.string.search_bar_hint, title)
     val sortModes = stringArrayResource(id = R.array.download_sort_modes)
     val downloadStates = stringArrayResource(id = R.array.download_state)
-    val list = remember(label, filterType, keyword, sortMode) {
+    val list = rememberInVM(filterState, invalidateKey) {
```

Comment:
Why we use `rememberInVM` here?
DownloadList maybe changed when user navigate (DownloadScreen -> GalleryDetailScreen -> GalleryListScreen) and we have no mechanism to invalidate it(unlike DAO `PagingSource`)

Reply:
Fair.
I was considering we want to keep it across configuration changes, but in reality users won't do that frequently.